src/components/dataTable/icon.scss

```diff
@@ -0,0 +1,37 @@
+table.md-table thead th {
```

Comment:
I did this to separate the overrides for the `md-icon` directive, same with `md-checkbox`. I thought it was cleaner.